wbb/modules/admin.py

```diff
@@ -1,14 +1,20 @@
 from pyrogram.types import Message
 from wbb import OWNER_ID, SUDO_USER_ID, app
 from wbb.utils import cust_filter
+from wbb.utils.botinfo import BOT_ID
 
 __MODULE__ = "Admin"
 __HELP__ = "/ban    - Ban A User\n" \
            "/unban  - Unban A User\n" \
            "/kick   - Kick A User\n" \
            "/purge  - Purge Messages\n" \
+           "/del    - Delete Replied Message\n" \
            "/banme  - Bans A User Who Issued The Command\n" \
-           "/kickme - Kicks A User Who Issued The Command"
+           "/kickme - Kicks A User Who Issued The Command\n" \
+           "/demote - Demote A Member\n" \
+           "/promote - Promote A Member\n" \
+           "/pin - Pin A Message\n" \
+           "/unpin - Unpin A Message"
 
```

Comment:
Use multiline string here

Reply:
For every file? or only this one?

Reply:
Every file maybe, for those ones which has multiple lines and each line is not too long.

Reply:
ok